### ISSUE TITLE
Suggestion! Per Vault Perms

### ISSUE THREAD

**Issue by luridftw32** [drtshock/PlayerVaults#99]
A way to create permission nodes per vault. I run Factions Grief and Nogrief on 1 server so it would be nice to be able to give permissions for the first 6 in Grief, then only perms 7-12 in NoGrief.

I also have a ranking system in which the higher rank you are the more vaults you can use. I have tried to block the commands in these worlds with World Guard but ran into conflictions with my ranks.

**Comment by drtshock** [comment]
You can just give them playervaults.amount.# and playervaults.size.#

**Comment by luridftw32** [comment]
If I give them playervaults.amount.12 for example it gives them all 12. I need my Member Rank to use /PV 1 in one world and /PV 7 in another. My VIP Rank uses 1-6 in one world and 7-12 in another.
However I found a plugin that can alias plugins and create a permission node for it, So I just aliased /pv 1 as /pv 1 to create a new node for it. Doing it 3 more times for all the aliases playervaults has.
